src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -91,7 +91,7 @@ public void testParallelPutRequests() throws Exception {
                 Assert.assertFalse(created);
                 created = true;
             } else {
-                Assert.assertEquals(HttpStatus.SC_CONFLICT, sc);
+                Assert.assertTrue("Status code should be 409 or 200", sc == HttpStatus.SC_CONFLICT || sc == HttpStatus.SC_CREATED);
```

Comment:
This is not a correct condition (should be `SC_OK` instead of `SC_CREATED`). IMO, it will be better not to mix 409 and 200 codes together, consider using `switch(sc)` with `SC_CREATED`, `SC_OK` and `default` cases.

Reply:
Yes, `SC_OK` should be seen only after `SC_CREATED`. `SC_CONFLICT` will be in the `default` case. Fixed.

Reply:
I was not referring to the order of `SC_OK`, especially that order in which requests are processed is not necessarily the same as `requests` order.